Subgraph isomorphism: readme information is added (#1664)
oneapi-src/oneDAL: README.md

	- Jaccard Similarity Coefficients for all pairs of vertices, a batch algorithm that processes the graph by blocks
  - Local and Global Triangle Counting
  - Single Source Shortest Paths (SSSP)

>   - Subgraph isomorphism algorithm for induced and non-induced subgraph for undirected graphs. Supports integer vertex attributes. Current version does not support edge attributes and directed graphs.

Reviewer: ```suggestion
  - Subgraph isomorphism algorithm for induced and non-induced subgraphs in undirected graphs (integer vertex attributes are supported, edge attributes are not supported).
```
